frontend/src/js/controllers/authCtrl.js

```diff
@@ -320,9 +320,9 @@
                         vm.getUser.email = '';
                         vm.stopLoader();
                     },
-                    onError: function() {
+                    onError: function(response) {
                         vm.isFormError = true;
-                        vm.FormError = "Something went wrong. Please try again";
+                        vm.FormError = (response.status == 400) ? response.data.details : "Something went wrong. Please try again";
```

Comment:
Do all the errors from ValidationError have 400 status and will send `response.data.details`?

Reply:
ValidationError sends the status 400 by default for bad request unless overridden. i have included a details message for both ValidationError so it will send it with all type of errors.

Reply:
Basically, if any other arises, earlier we were just saying `Something went wrong. Please try again`, but now it will show the details for other errors as well (including Inactive Account)?

Reply:
No it only covers if the account is inactive  or user doesn't exist, for other than these type of issues, it will resolve to something is  wrong message. if status code is not 200 or 400.